scripts/ci-smoketest.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+# Travis CI Release / Smoketest Script
+#
+LAYER0_PATH=$GOPATH/src/github.com/quintilesims/layer0
+LAYER0_PREFIX=smoketest
+set -e
+set -x
+
+# Ignore if master branch is not target
+if [ ! "$TRAVIS_BRANCH" == "master" ]; then
+  echo "[INFO] Skipping Smoketest";
+  exit 0
+fi
+
+# Tag repository
+git tag "$LAYER0_PREFIX"
```

Comment:
re: Travis keeping git data, there are multiple bugs out for it. Some people have tried preventing Travis' "shallow clone" by setting the `depth` to 99999 and/or all sorts of other weird workarounds. More time could be spent exploring this, but the bottom line is that the behavior is non-standard. 

One idea I had was to update a `git tag` variable as part of Travis' global env vars when we run a release. Another would be to set the tag in the `travis.yml` file prior to a release. Essentially it's hardcoded for now, and I feel that we could refactor this part when we add the release automation.